buildspec-windows.yml

```diff
@@ -64,6 +64,9 @@ phases:
       - |
         $env:Path = "C:\tools\cygwin\bin;$env:Path"
         cmd /c 'call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x64 && bash -c "make -C regression test BUILD_ENV=MSVC" '
+
+      - |
```

Comment:
Apologies if I don't understand the buildspec + windows combination - but why does `regression/goto-cl` need to be treated any differently from, any of the other subdirectories in `regression/` ?

Reply:
Fixing up the `make test` target to understand that on win32 or whatever, extra targets should be added would probably be preferable/less error prone when moving between CI infrastructure, etc.

Reply:
I'll take care of this in a separate PR. We also have some weird interaction between scripts removing dirs and the makefiles at the moment, which should all go away thanks to the -X/-I options in test.pl.